src/test/resources/Homework19.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE suite SYSTEM "http://testng.org/testng-1.0.dtd" >
+
+<suite name="Homework19">
+
+    <parameter name="BaseURL" value="https://bbb.testpro.io/" />
+
+    <test name="Example Test" preserve-order="false">
+        <classes>
+            <class name="BaseTest"/>
+            <class name="Homework19"/>
+        </classes>
+    </classes>
+</test>
```

Comment:
You got this problem, please correct it. 

![image](https://user-images.githubusercontent.com/104761482/215258410-ef639aa6-edff-4791-982c-7fcebf650aeb.png)

For example 

```
<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE suite SYSTEM "http://testng.org/testng-1.0.dtd" >

<suite name="Homework19">
    <parameter name="BaseURL" value="https://bbb.testpro.io/" />
    <test name="Example Test" preserve-order="false">
        <classes>
            <class name="BaseTest"/>
            <class name="Homework19"/>
        </classes>
    </test> 
</suite>
```